api.go

```diff
@@ -641,7 +654,8 @@ func KillContainer(podID, containerID string, signal syscall.Signal, all bool) e
 	}
 
 	// Send a signal to the process.
-	err = c.kill(signal, all)
+	cImpl := c.(*Container)
```

Comment:
Not needed if we agree that `fetchContainer()` should not return `VCContainer` type.